src/pytorch_lightning/plugins/precision/sharded_native_amp.py

```diff
@@ -22,6 +22,8 @@
 if _FAIRSCALE_AVAILABLE:
     from fairscale.optim import OSS
     from fairscale.optim.grad_scaler import ShardedGradScaler
+else:
+    OSS = ShardedGradScaler = object
```

Comment:
Instead of these changes, you need to update the mypy files attribute in pyproject.

Reply:
not really, this was crashing building docs